customize.dist/translations/messages.es.js

```diff
@@ -18,7 +18,7 @@ define(function () {
     out.synchronizing = "Sincronización";
     out.reconnecting = "Reconectando...";
     out.lag = "Retraso";
-    out.readonly = 'Solo lectura';
+    out.readonly = 'Sólo lectura';
```

Comment:
pbondoer:  ~~almost~~ impossible. I work on a company who makes internationalization on a daily basis (but mainly for Subtitles, Transcripts, etc). The rule **here** is going for the variation of the language with the bigger population worldwide. I'm saying this just as a point of comparison.